### PR TITLE
make initialization of wings simpler and make it easier to override

### PR DESCRIPTION
**Pull request by orangewolf**
By setting `WINGS_CONCERNS` and using it we can make it simpler to set Wings and we can then use that set up to look them back up in reverse for the valkyrie adapter.

### CODE REVIEW

File: config/initializers/wings.rb [hunk 1]
```diff
@@ -110,34 +105,8 @@
...
-      Hyrax::AccessControl
-    elsif 'FileSet' == klass_name
-      Hyrax::FileSet
-    elsif 'Hydra::AccessControls::Embargo' == klass_name
-      Hyrax::Embargo
-    elsif 'Hydra::AccessControls::Lease' == klass_name
-      Hyrax::Lease
-    elsif 'Hydra::PCDM::File' == klass_name
-      Hyrax::FileMetadata
-    else
```
Comment (laritakr) [inline config/initializers/wings.rb:138]: I'm a bit concerned with the removal of the above mappings. Where does Hydra::AccessControls::Embargo get mapped to Hyrax::Embargo, etc? All of these were added due to prior errors, and I want to be sure we aren't breaking prior fixes.

Comment (orangewolf) [reply config/initializers/wings.rb:138]: They are initialized in Hyrax wings adapter. If you open the Hyku console and do Wings::ModelRegistry.reverse_lookup(Hydra::AccessControls::Embargo) you will see the right result

Comment (orangewolf) [reply config/initializers/wings.rb:138]: Or at least I did when I tried them all one by one

**Review by laritakr — APPROVED** [review]

File: config/initializers/wings.rb (hunk 1 above)
Comment (laritakr) [reply config/initializers/wings.rb:138]: I tried a few and they worked so I approved.

**Pull request merged**